radio/src/targets/common/arm/stm32/sdio_sd.c

```diff
@@ -54,7 +54,12 @@
 #define SD_STD_CAPACITY                 ((uint32_t)0x00000000)
 #define SD_CHECK_PATTERN                ((uint32_t)0x000001AA)
 
-#define SD_MAX_VOLT_TRIAL               ((uint32_t)0x0000FFFF)
+#if defined(RADIO_TX16S)
+  #define SD_MAX_VOLT_TRIAL               ((uint32_t)0x00000834)
```

Comment:
Is there a reason why 2100 was chosen? An explaining comment for this magic number would be sweet. Thanks!

Reply:
We have gone through a lot of testing, constantly turning the Radio on and off to reproduce the problem of it not being able to start. Sometimes it happens after approximately 200 restarts, and sometimes after 30-50. Each time, we inspect the debug information through serial port connection and find that it gets stuck at startup. The remote control will attempt 65,536 times during startup, taking about one to two minutes. This has caused distress for users, so we tried to reduce this number.

2100 is a figure we obtained from others' shared experiences. We chose it after extensive testing. 2100 times saves a lot of time compared to 65,536 times. We also tried reducing the number to 500 and increasing it to 8000. When the value is very low, like 500, there are occasional issues with the splash image not displaying. When the value is larger, it takes a few seconds or even tens of seconds longer.

After thousands of on-off cycle tests, we decided to use 2100. When the SD card encounters an error during startup, it won't make users feel like the Radio is broken and remove the battery to restart. In fact, waiting one or two minutes would allow it to start, but because users are not aware of this phenomenon, they would immediately remove the battery and restart the Radio. Then another issue arises. Removing the battery has a high probability of causing RTC Backup memory data errors. After starting again, it would still be a black screen, and users would need to remove the button battery on the motherboard to reset, a symptom we have reproduced multiple times.

Now, a simple problem has led to the emergence of another complex one, causing distress for many users. So, we tried to modify this number to prevent the Radio from being unable to start. Practice has proved this to be very effective. Moreover, I saw another PR #3576, adding CRC check to RTC Backup data to avoid the problem of not being able to start. I believe these modifications and optimizations can solve the problem that has been bothering many people for a long time.

Reply:
@zyren Hi, have you tested #3576? Do you think clearing RTC->BKP0R and  RTC->BKP1R in case of the "CRC" failing (it's not really a CRC but I believe good enough for a starter) is the right way? Can you provide test cases (inconsistent BKP0R/1R value combinations) which will force a TX16s to be stuck at boot?

Reply:
> @zyren Hi, have you tested #3576? Do you think clearing RTC->BKP0R and RTC->BKP1R in case of the "CRC" failing (it's not really a CRC but I believe good enough for a starter) is the right way? Can you provide test cases (inconsistent BKP0R/1R value combinations) which will force a TX16s to be stuck at boot?

I'm glad you're excited about this pull request (PR #3576). Before this, you've encountered similar issues where the radio couldn't be powered on, but removing the RTC battery for a while and then reinstalling it resolved the issue. Therefore, you plan to extensively test this PR to draw conclusions. It may take a few days to validate because these issues are difficult to reproduce and require patience and time.